website/source/docs/providers/google/d/datasource_compute_network.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "google"
+page_title: "Google: google_compute_network"
+sidebar_current: "docs-google-datasource-compute-etwork"
+description: |-
+  Get a network within GCE.
+---
+
+# google\_compute\_network
+
+Get a network within GCE from his name.
+
+## Example Usage
+
+```js
```

Comment:
You can use tf for the language now! (I think this is a relatively new development)